fbw-common/src/wasm/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -1829,6 +1847,16 @@ impl LinearActuatedRigidBodyOnHingeAxis {
         }
     }
 
+    fn limit_absolute_angular_speed(&mut self) {
+        let max_angular_speed =
+            AngularVelocity::new::<radian_per_second>(Self::MAX_ABSOLUTE_ANGULAR_SPEED_RAD_S);
+
+        self.angular_speed = self
+            .angular_speed
+            .min(max_angular_speed)
+            .max(-max_angular_speed);
```

Comment:
```suggestion
            .angular_speed
            .clamp(-max_angular_speed, max_angular_speed);
```
You introduced this function in one of your previous PRs :smile: